src/api/comments.ts

```diff
@@ -0,0 +1,37 @@
+export const BASE_URL = 'https://mate.academy/students-api/';
+
+export async function getComments(postId: number) {
+  const response = await fetch(`${BASE_URL}/comments?postId=${postId}`);
+
+  if (!response.ok) {
+    throw new Error(`${response.status} - ${response.statusText}`);
+  }
+
+  return response.json();
+}
+
+export async function delComment(id : number) {
```

Comment:
```suggestion
export async function deleteComment(id: Comment['id']) {
```
Краще не скорочувати назви змінних, наскільки це можливо
Зайвий пробіл між `id` та `:`